openedx/adg/lms/applications/models.py

```diff
@@ -30,25 +36,57 @@ def __str__(self):
         return 'User {user_id}, application status id={id}'.format(user_id=self.user.id, id=self.id)
 
 
+class BusinessLine(TimeStampedModel):
+    """
+    Model to save the business lines
+    """
+    title = models.CharField(verbose_name=_('Title'), max_length=255, unique=True, )
+    logo = models.ImageField(
+        upload_to='business-lines/logos/', null=True, verbose_name=_('Logo'),
+        validators=[FileExtensionValidator(ALLOWED_LOGO_EXTENSIONS), validate_logo_size],
+    )
+    description = models.TextField(verbose_name=_('Description'),)
+
+    def __str__(self):
+        return 'Business Line {}'.format(self.title)
+
+
 class UserApplication(TimeStampedModel):
     """
     Model for status of all required parts of user application submission.
     """
-    user = models.OneToOneField(User, related_name='application', on_delete=models.CASCADE, )
+    user = models.OneToOneField(User, related_name='application', on_delete=models.CASCADE, verbose_name=_('User'), )
     city = models.CharField(verbose_name=_('City'), max_length=255, )
+
+    saudi_national = models.BooleanField(default=True, verbose_name=_('Saudi National'),)
+    MALE = 'male'
+    FEMALE = 'female'
+    OTHER = 'other'
+
+    GENDER_CHOICES = (
+        (MALE, _('Male')),
+        (FEMALE, _('Female')),
+        (OTHER, _('Prefer not to answer')),
+    )
+    gender = models.CharField(verbose_name=_('Gender'), choices=GENDER_CHOICES, max_length=8, default=OTHER,)
```

Comment:
@ahmed-faraz46 @zahid-arbisoft @zeshanarifios We do have a gender field in UserProfile, Also why we are adding profile related attributes like **gender**, **saudi_national**, **phone_number**, **birth_date** in application model, We should look into the core UserProfile and use whatever we can use from that model and If we cannot use that model then we can add these profile attributes to another model which we added for ADG, named ExtendedUserProfile? What do you all suggest?

Reply:
I agree, unless there is a specific requirement to have different `gender, saudi_national, phone_number, birth_date`

Reply:
Good suggestion @asadali145 I agree with you, we should keep profile related data in profile. So if there is any existing field in UserProfile we should use that, if there is no existing field, we should add that field in `UserExtendedProfile`. I have discussed with @zahid-arbisoft he agrees, I also discussed with @harris-irfan he also agrees and according to him we can tackle this in our admin side as well.
So @javeriaejaz-arbisoft let's please proceed with it.

Reply:
@zahid-arbisoft @ahmed-faraz46 ExtendedUserProfile is currently using User from django.contrib.auth.models which doesn't have gender field. If we are going to use UserProfile from edx-platform/common/djangoapps/student/models.py then we have to change ExtendedUserProfile accordingly. Also UserProfile have many unwanted fields.

Reply:
@javeriaejaz-arbisoft gender is in core UserProfile and that is connected with Auth User model as well and we have added this new model to keep our adg related fields for a user and to avoid core changes. This new Extended profile is also connected with Auth user model, We can access both profiles from User object. Leave the extra fields that are in UserProfile. @ahmed-faraz46 right?

Reply:
Having a link/relation with the user model is enough I guess